### ISSUE TITLE
Shortcut details in the App

### ISSUE THREAD

**Issue by khurram-hassan** [Genymobile/scrcpy#1674]
- [x] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

Hi,
I like this application very much. I love using it all the time when I am working from my PC or laptop. It charges my phone simultaneously while I work on both PC and my phone _(through your app)_. 

**Is your feature request related to a problem? Please describe.**
However, I would like to suggest a feature in the application which frustrates me a bit. There is no way of knowing the shortcut keys and other useful features of the App from inside the app itself.

**Describe the solution you'd like**
It would be good if you can show the shortcuts and other App related details or other useful features of the App from within the App itself somehow. 

**Describe alternatives you've considered**
_(Workaround)_ Right now if I need to find a shortcut I have to open the **scrcpy** Github page. For example recently you have changed the Modifier key from 'Ctrl' to 'LAlt' and I was confused why some of the shortcuts stopped working while it was working on earlier builds. But when I looked at the shortcut details carefully, I understood that the modifier keys have been changed.

**Additional context**
Sample screenshot of how it could be possible. I am including a screenshot of an emulator where I play PUBG and it shows me all the possible shortcuts beside the Emulator window. This is also collapsible. And a user who is accustomed to these may want to collapse this area. I believe this is a feature which would bring more users to use this app and retain them to this app due to the simplicity. Right now, some users may find it difficult to navigate within the app without proper guidelines.
![Emulator example](https://user-images.githubusercontent.com/68346553/90235849-97d75900-de3f-11ea-9e8a-1dc8322cd863.png)

I hope you understand the importance of this feature. Looking forward to see this feature in upcoming releases.

Thanks.

**Comment by rom1v** [comment]
Yes, but it's not simple in SDL which does not provide UI widgets.

So it would require another UI framework (Qt?). I "plan" (it's more a "wish" than a "planning") to refactor the code to split the core and UI in a way that allow to write another UI (using another framework) while using the same core. This would be the first step…

_Just for info (I know this is not what you are requesting), the help is also available via `man scrcpy` or `scrcpy --help`._

**Comment by automatata** [comment]
By the way, for exactly the purpose I have a little script `show-scrcpy-shortcuts.bat` for my own use:

```bat
@echo off
mode con: cols=80 lines=36
type shortcuts.txt
echo.
@pause
```

together with a manually edited `shortcuts.txt` listing the current shortcuts:

```
Action                                      |   Shortcut
--------------------------------------------+----------------------------
Switch fullscreen mode                      | MOD + f
Rotate display left                         | MOD + left
Rotate display right                        | MOD + right
Resize window to 1:1 (pixel-perfect)        | MOD + g
...
...
```

The script shows a console window printing the above content, which is dismissed conveniently by any key press. Then one can make a shortcut to launch `show-scrcpy-shortcuts.bat` and pin it in the start menu or whatever convenient place. One can further assign a keyboard hotkey to launch it. The script is for Windows but in other OSes you can do similar things.

The line `mode con: cols=80 lines=36` is optional and sets the console window to a size fitting the text file. The line count `36` should be updated to (at least) your `shortcuts.txt`'s line count plus 2. And of course when scrcpy is updated you may have to edit `shortcuts.txt` as well.

Extra: I have installed Git bash which provides the `wc` command, so I actually replaced `mode con: cols=80 lines=36` by

```bat
for /f "delims=" %%i in ('wc -l shortcuts.txt ^| cut -d ' ' -f 1') do (set /a lines=%%i+2)
mode con: cols=80 lines=%lines%
```

which automates the line count. There should be other ways without the `wc` command, too. Anyway, even if you hard code the line count, it's not much work to update it once in a while when scrcpy has updated the shortcuts.

**Comment by shripal17** [comment]
Hello all,

I have made a plugin for IntelliJ IDEA/Android Studio users:
https://github.com/shripal17/ScrcpyController
https://plugins.jetbrains.com/plugin/14565-scrcpy/

One feature of the plugin is:
It fetches scrcpy's README, extracts the shortcuts section from it, and displays it in HTML format.

But this plugin is designed only for working within IntelliJ/Android Studio,

If there are people who would like this plugin to work independently, I can think of making it separately in swing
